terraform/lb.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(icco): This is currently all setup manually.
```

Comment:
Are you saying our Terraform setup isn't currently managing this? We should probably merge as-is and then import the existing state. I can help with that.

Reply:
A mixture, the uncommented things we'll need to import. The commented things are currently broken in google-beta. https://github.com/hashicorp/terraform-provider-google/issues/7051 will fix it in the next release.